src/commonMain/resources/db/migration/V41__BookmarksIndex.sql

```diff
@@ -0,0 +1,5 @@
+CREATE INDEX IF NOT EXISTS playlist_type
+    ON public.playlist USING btree
+        ("createdAt" ASC NULLS LAST, "type")
+    TABLESPACE pg_default
+    WHERE "type" != 'Private' AND "deletedAt" IS NULL;
```

Comment:
Need to add an index so that the insert ignore works:
```sql
CREATE UNIQUE INDEX IF NOT EXISTS bookmark_playlist
    ON public.playlist USING btree ("owner")
    WHERE "type" = 'System';
```